src/components/Navbar/NavTabs.js

```diff
@@ -35,26 +91,29 @@ function NavTabs() {
 
   const renderListItems = (path) => {
```

Comment:
Can you give an explanation or comment in the code about what is happening in this function?

Reply:
@rishFilet I will write a comment for this code from my branch since I wrote it.